docker/bookkeeper/Dockerfile

```diff
@@ -1,12 +1,17 @@
 #
-# Copyright (c) 2017 Dell Inc., or its subsidiaries. All Rights Reserved.
+# Copyright (c) Dell Inc., or its subsidiaries. All Rights Reserved.
 #
 # Licensed under the Apache License, Version 2.0 (the "License");
 # you may not use this file except in compliance with the License.
 # You may obtain a copy of the License at
 #
 #     http://www.apache.org/licenses/LICENSE-2.0
 #
+# This file contains source code copied from the Apache BookKeeper project (http://bookkeeper.apache.org).
```

Comment:
Could you separate this comment block from the license header block? Also, break the line earlier, like I mentioned in the previous comment.

Reply:
Sure, done.